contrib/static-builder-x86_64/Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM nixos/nix
```

Comment:
don't we still need this file?

Reply:
No we don’t, since this PR is now based on official nixos/nix docker image directly without modification.

The use of official nixos/nix is just for simplify the installation of nix and nix-build, by copying the /nix from image to localhost before running test.

Reply:
See https://github.com/containers/crun/pull/428/files#diff-e8f2acbca1a8e793c628242f3f934107R39-R49